clusters/snd/01/kustomization.yaml

```diff
@@ -0,0 +1,2 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1
```

Comment:
I am generally happy with this PR, but would like some stuff added to the readme to detail what this repos generally about and what it contains @ppotturi @kbabigumira

Reply:
Readme updated